_posts/2020-04-28-history-of-bel.md

```diff
@@ -35,30 +35,72 @@ community. And so, we being on a dark and stormy night in 2001....
 
 ### 2001
 
-**2001** Selventa is [founded](https://www.flagshippioneering.com/companies/selventa) by Keith Elliston (originally as GenStruct, Inc.)
+**2001** Selventa is [founded](https://www.flagshippioneering.com/companies/selventa) by [Noubar Afeyan](https://www.linkedin.com/in/noubarafeyan/) as Genstruct, Inc.
+The concept was to create a computational complement to the company [Beyond Genomics, Inc.](https://bg-medicine.com) - which was a systems biology company also founded by AGTC (a predecessor fund to Flagship).
```

Comment:
do you have a link for AGTC, or a link to an article describing its evolution to Flagship?